Update authentication.md

Sergey-Shabanov opened the pull request:
Привидение authentication.md к последнему коммиту e2343d1fe91a2c36e7a6b220aff995c1f48a8405 +
Правка url до локальных файлов документации (в оригинальной такой же косяк имеется) +
Правка смысловых и орфорграфических конструкций

Sergey-Shabanov committed:
Update authentication.md

Привидение authentication.md к последнему коммиту e2343d1fe91a2c36e7a6b220aff995c1f48a8405 +
Правка url до локальных файлов документации (в оригинальной такой же косяк имеется) +
Правка смысловых и орфорграфических конструкций

slider23 commented:
`/docs/{{version}}/` вырезать нельзя, это конструкции, необходимые для формирования ссылок. Этот перевод же потом выкладывается на сайт.

Sergey-Shabanov commented:
> `/docs/{{version}}/` вырезать нельзя, это конструкции, необходимые для формирования ссылок. Этот перевод же потом выкладывается на сайт.

Я не правильно понимаю, что документация на github в первую очередь для github, а уже во вторую для сайта?
Я думал, что сайт сам резолвит ссылки (дополняет).
Сёрфить по репе перевода становится тем ещё квестом...
Ок, сейчас это недоразумение поправлю.

Sergey-Shabanov committed:
Update authentication.md

Вернул конструкцию /docs/{{version}}/ которую удалил в предыдущем коммите.
Указанная конструкция используется для формирования ссылок на сайте.

Sergey-Shabanov commented:
> > `/docs/{{version}}/` вырезать нельзя, это конструкции, необходимые для формирования ссылок. Этот перевод же потом выкладывается на сайт.
> 
> Ок, сейчас это недоразумение поправлю.

Поправил

Sergey-Shabanov committed:
Merge branch 'LaravelRUS:8.x' into 8.x
